### PR TITLE
Tag Plots.jl v0.19.3

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JuliaPlots/Plots.jl](https://github.com/JuliaPlots/Plots.jl)
Release: [v0.19.3](https://github.com/JuliaPlots/Plots.jl/releases/tag/v0.19.3)
Travis: [Travis Build Status](https://travis-ci.org/JuliaPlots/Plots.jl/builds/417396819)
Diff: [vs v0.19.2](https://github.com/JuliaPlots/Plots.jl/compare/2d217edc157afda5112992b82d9d2b707b8b6610...60b8b0080cdfd1b2945b95b0f8ff41e0932af59b)
`requires` vs v0.19.2: no changes
cc: @mkborregaard

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds reflect minimum requirements.

### CODE REVIEW

**Comment by vchuravy** [comment]
@mkborregaard Test are failing on travis

**Comment by mkborregaard** [comment]
@vchuravy yes they have been failing on 1.0 since the release. That's because our testing pipeline involves saving plotted figures to png, loading them, and comparing them to a continuously updated reference database, so we can track any changes that lead to differences in the output figure. This process requires Images and ImageMagick which are not yet 1.0 compliant. So since the release of 0.7 we've been manually working through all of our test plots and inspected them for regressions and errors.
In short it works fine and has done since v0.19.0. The three patch versions we've released since then are testimony that our test suite apparently isn't covering all corner cases and needs to be expanded. Fortunately people report errors on Plots very quickly.
